config/ember-try.js

```diff
@@ -1,11 +1,36 @@
 /*jshint node:true*/
 module.exports = {
-  useVersionCompatibility: true,
```

Comment:
I know that it's pulled for ember-observer to show us this table:

![screen shot 2017-01-30 at 11 31 35 am](https://cloud.githubusercontent.com/assets/630449/22436054/bb7631c0-e6df-11e6-9cfb-c35a03784ab3.png)

but I think since the default `ember init` isn't including that versionCompatiblity flag, we can remove it and stay in-sync with the community.